src/main/java/com/mabl/integration/jenkins/MablSystemException.java

```diff
@@ -0,0 +1,20 @@
+package com.mabl.integration.jenkins;
+
+/**
+ * Non test related failure
+ *
+ */
+class MablSystemException extends Exception {
```

Comment:
Do we expect people to do anything these? If the exceptions just bubble up through code to top level handler, you may want to use RuntimeException here so you do not have to re-throw them all the way up the call stack.